package/lang/Default.ini

```diff
@@ -130,7 +130,7 @@ res_large_text_AA=Large text %s at Level AA
 res_large_text_AAA=Large text %s at Level AAA
 
 lcr_ratio_is=The contrast ratio is: %s
-contrast_ratio=Contrast ratio: %s
+just_below=The contrast ratio is: just below %s
```

Comment:
this will presumably need to be added (in translated form) to all other language versions?